packages/table/src/table.tsx

```diff
@@ -386,12 +386,11 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     }
 
     public renderHotkeys() {
-        let hotkeys = [this.maybeRenderCopyHotkey(), this.maybeRenderSelectAllHotkey()];
-        // remove null elements
-        hotkeys = hotkeys.filter((element) => element != null);
+        const hotkeys = [this.maybeRenderCopyHotkey(), this.maybeRenderSelectAllHotkey()];
         return (
             <Hotkeys>
-                {hotkeys}
+                {/* filter to remove null elements */}
```

Comment:
I mean, I kindof wanted to say _why_ I need to do that. But yeah, the comment is kindof useless.